tardis/apps/publication_workflow/components/Forms/SelectDatasetForm.jsx

```diff
@@ -0,0 +1,215 @@
+              }
+              isInvalid={!!formik.errors.publicationDescription}
+            />
+            <Form.Control.Feedback type="invalid">
+              {formik.errors.publicationDescription}
+            </Form.Control.Feedback>
+          </Form.Group>
+          <Row>
+            <Col className="col-md-6">
+              <Form.Group className="mb-3" controlId="formGroupExperiment">
+                <Form.Label>Select experiment</Form.Label>
+                <Form.Control
+                  as="select"
+                  name="experimentDropDown"
+                  onChange={handleExpChange}
+                >
+                  {expList.map((value, index) => (
+                    <option
+                      value={index}
+                      key={value.id}
+                    >
+                      {value.title}
+                    </option>
+                  )) }
+                </Form.Control>
+              </Form.Group>
+              <Form.Group className="mb-3" controlId="formGroupDatasets">
+                <Form.Label>Select Datasets</Form.Label>
+                <div className="row">
+                  <div className="form-horizontal col-md-12">
+                    <div className="form-group has-search">
+                      {/* <span className="fa fa-search form-control-feedback"/> */}
+                      <input
+                        type="text"
+                        className="form-control"
+                        placeholder="Just start typing to filter datasets based on description"
+                        onKeyUp={onFilter}
+                      />
+                    </div>
+                  </div>
+                </div>
+                <Form.Control
+                  as="select"
+                  multiple
+                  onChange={handleDatasetSelectChange}
+                >
+                  {datasetList.map(value => (
+                    <option
+                      value={value.id}
+                      key={value.id}
+                    >
+                      {value.description}
+                    </option>
+                  )) }
+                </Form.Control>
+              </Form.Group>
+            </Col>
+            <Form.Control
+              as="input"
+              type="hidden"
+              name="selectedDatasets"
+              value={formik.values.selectedDatasets}
+              onChange={formik.handleChange}
+              isValid={formik.touched.selectedDatasets
+              && !formik.errors.selectedDatasets
+              }
+              isInvalid={!!formik.errors.selectedDatasets}
+            />
+            <Col className="col-md-6">
+              <h4>Added Datasets</h4>
+              <Table size="sm" responsive striped hover>
+                <thead>
+                  <tr>
+                    <th>Experiment</th>
+                    <th>Description</th>
+                    <th />
+                  </tr>
+                </thead>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 32 spaces but found 16.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                                </thead>
```